src/main/java/tc/oc/pgm/maptag/StandardMapTags.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Aleksander Jagiełło <themolkapl@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tc.oc.pgm.maptag;
+
+import static com.google.common.base.Preconditions.*;
+
+import java.util.function.Predicate;
+import tc.oc.pgm.blitz.BlitzModule;
+import tc.oc.pgm.classes.ClassModule;
+import tc.oc.pgm.controlpoint.ControlPointModule;
+import tc.oc.pgm.core.CoreModule;
+import tc.oc.pgm.crafting.CraftingModule;
+import tc.oc.pgm.destroyable.DestroyableModule;
+import tc.oc.pgm.ffa.FreeForAllModule;
+import tc.oc.pgm.flag.FlagModule;
+import tc.oc.pgm.hunger.HungerModule;
+import tc.oc.pgm.map.MapModule;
+import tc.oc.pgm.map.PGMMap;
+import tc.oc.pgm.modules.InfoModule;
+import tc.oc.pgm.modules.InternalModule;
+import tc.oc.pgm.modules.LaneModule;
+import tc.oc.pgm.rage.RageModule;
+import tc.oc.pgm.score.ScoreModule;
+import tc.oc.pgm.teams.TeamModule;
+import tc.oc.pgm.terrain.TerrainModule;
+import tc.oc.pgm.timelimit.TimeLimitModule;
+import tc.oc.pgm.tnt.TNTModule;
+import tc.oc.pgm.wool.WoolModule;
+import tc.oc.pgm.worldborder.WorldBorderModule;
+
+public interface StandardMapTags {
+  StandardMapTag _4TEAMS = create("4teams", TeamModule.class, team -> team.getTeams().size() == 4);
+  StandardMapTag BLITZ = create("blitz", BlitzModule.class, blitz -> !blitz.isDisabled(null));
+  StandardMapTag CLASSES = create("classes", ClassModule.class);
+  StandardMapTag CORE = create("core", CoreModule.class);
+  StandardMapTag CP = create("cp", ControlPointModule.class);
+  StandardMapTag CRAFTING =
+      create("crafting", CraftingModule.class, crafting -> !crafting.getCustomRecipes().isEmpty());
+  StandardMapTag DESTROYABLE = create("destroyable", DestroyableModule.class);
```

Comment:
no one calls them destroyables, `#monument` ?

Reply:
DTE (Destroy The Enemy) uses destroyables to define islands. If we aren't calling them `#destroyable` I would go for `#dtm` instead.

Reply:
You have `#core` and `#wool`, basically missing `#monument`, otherwise, rename them all to `#dtc`, `#ctw`, `#dtm`, but those names may be a more cryptic for users.

Destroy the enemy is just a type of map that uses a big monument, but it's still a monument. Code-wise they're called destroyables, but they're only called that to the user when it's destroy the enemy, and never on any other monument map (eg: warlock, snowy wars).